_data/definitions.yml

```diff
@@ -1,17 +0,0 @@
-# Password screen
-
-changepassword_password: >
- <p>This is a sample list of stuff:<br/>
- - more stuff<br/>
- - another line<br/>
- </p>
-
-elephant: "This is a sample definition."
-
-baseball: "Baseball is considered America's pasttime sport, though that may be more of a historical term than a current one. There's a lot more excitement about football than baseball. A baseball game is somewhat of a snooze to watch, for the most part."
-
-basketball: "Basketball is a sport involving two teams of five players each competing to put a ball through a small circular rim 10 feet above the ground. Basketball requires players to be in top physical condition, since they spend most of the game running back and forth along a 94-foot-long floor."
-
```

Comment:
Haha. This term means something very specific in the context of CockroachDB ;).

Not sure how this and the `glossary.yml` file differ. I think they're both included in the theme so that it's possible to store definitions in a central place and then pull them into a glossary page or use them as tooltips. But feel free to remove and we can reintroduce differently later, if needed.